Softeq.XToolkit.WhiteLabel/Navigation/NavigateHelper.cs

```diff
@@ -8,40 +9,43 @@ namespace Softeq.XToolkit.WhiteLabel.Navigation
 {
     public class NavigateHelper<TViewModel> where TViewModel : IViewModelBase
     {
-        private readonly TViewModel _viewModel;
-        private readonly Action<bool> _navigateAction;
+        private readonly Action<bool, IReadOnlyList<NavigationParameterModel>> _navigateAction;
+        private readonly List<NavigationParameterModel> _parameters = new List<NavigationParameterModel>();
 
-        public NavigateHelper(TViewModel viewModel, Action<bool> navigateAction)
+        public NavigateHelper(Action<bool, IReadOnlyList<NavigationParameterModel>> navigateAction)
         {
-            _viewModel = viewModel;
             _navigateAction = navigateAction;
         }
 
         public NavigateHelper<TViewModel> WithParam<TValue>(Expression<Func<TViewModel, TValue>> property, TValue value)
         {
-            SetMemberValue(property.GetMemberInfo(), _viewModel, value);
-            
+            var parameter = new NavigationParameterModel { Value = value };

```

Comment:
need the short page on the wiki about using this method (parameters type constraints )

Reply:
added to readme